tests/unittests/Foundation/ReferenceFoundation/TestNSBundle.mm

```diff
@@ -44,7 +44,8 @@
     // #endif
 
     ASSERT_OBJCEQ([bundle pathForAuxiliaryExecutable:@"no_such_file"], nil);
-    ASSERT_OBJCEQ(bundle.appStoreReceiptURL, nil);
+
+    // WINOBJC: Remove test that no longer passes on reference platform
```

Comment:
could you leave the original code here, commented out? that way it's easier to compare with the reference